README.md

```diff
@@ -18,14 +18,16 @@ policy for the provided forms of distribution.
 
 ## Set up a node and/or baking on Ubuntu
 
+> :warning: Since [#570](https://github.com/serokell/tezos-packaging/pull/570), wizards' names were simplified.
+
 The simplest procedure to set up a node and/or baking instance is provided for Ubuntu.
 
 These commands will install everything necessary and start an interactive
 setup wizard:
 ```
 sudo add-apt-repository -yu ppa:serokell/tezos
 sudo apt-get install -y tezos-baking
-tezos-setup-wizard
+tezos-setup
 ```
 
```

Comment:
I would move the warning here, as above it's more "on the way" of new users.

Also to make things clearer I'd mention the name. IMO
```suggestion
> :warning: Since [#570](https://github.com/serokell/tezos-packaging/pull/570) this wizard was renamed from `tezos-setup-wizard`

```
should be enough